File: SgartIT.Net.Angular.Esempio1.Server/Models/PuntoVenditaDettaglio.cs
namespace SgartIT.Net.Angular.Esempio1.Server.Models;

public class PuntoVenditaDettaglio : PuntoVendita
{
    public string Email { get; set; } = string.Empty;
    public string Telefono { get; set; } = string.Empty;
    public bool Abilitato { get; set; }
    public DateTime DataUltimaModifica { get; set; }

    public List<Prodotto> Prodotti { get; set; } = [];
    public List<Servizio> Servizi { get; set; } = [];
    public List<Carta> Carte { get; set; } = [];
}

File: sgartit.net.angular.esempio1.client/src/app/models/punto-vendita-dettaglio.ts
import { Carta } from './carta';
import { Prodotto } from './prodotto';
import { PuntoVendita } from './punto-vendita';
import { Servizio } from './servizio';
export interface PuntoVenditaDettaglio extends PuntoVendita {
  email: string;
  telefono: string;
  abilitato: boolean;
  dataUltimaModifica: Date;

  prodotti: Prodotto[];
  servizi: Servizio[];
  carte: Carta[];
}
